File: GremsReliquary/src/main/java/gremsReliquary/relics/normal/Placeholder.java
package gremsReliquary.relics.normal;

import com.badlogic.gdx.graphics.Texture;
import gremsReliquary.GremsReliquary;
import gremsReliquary.util.TextureLoader;
import kotlinReliquary.relics.AbstractGremRelic;

public class Placeholder extends AbstractGremRelic {
    public static final String ID = GremsReliquary.makeID(Placeholder.class.getSimpleName());
    public static final Texture IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/Placeholder.png");
    public static final Texture OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/Placeholder.png");
    
    public Placeholder() {
        super(ID, IMG, OUTLINE, RelicTier.RARE, RelicType.NORMAL, LandingSound.CLINK);
    }
    
    // Description
    @Override
    public String getUpdatedDescription() {
        return DESCRIPTIONS[0];
    }
}

File: GremsReliquary/src/main/kotlin/kotlinReliquary/relics/normal/Placeholder.kt
package kotlinReliquary.relics.normal

import com.megacrit.cardcrawl.relics.AbstractRelic
import gremsReliquary.GremsReliquary
import gremsReliquary.util.TextureLoader
import kotlinReliquary.relics.AbstractGremRelic

class Placeholder : AbstractGremRelic(ID, IMG, OUTLINE, AbstractRelic.RelicTier.RARE, AbstractGremRelic.RelicType.NORMAL, AbstractRelic.LandingSound.CLINK) {

    companion object {
        val ID = GremsReliquary.makeID(Placeholder::class.java.simpleName)
        val IMG = TextureLoader.getTexture("gremsReliquaryResources/images/relics/Placeholder.png")
        val OUTLINE = TextureLoader.getTexture("gremsReliquaryResources/images/relics/outline/Placeholder.png")
    }


    override fun getUpdatedDescription(): String {
        return DESCRIPTIONS[0]
    }

}